source/MRPch/MRSuppressWarning.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#define MR_STR(...) #__VA_ARGS__
+
+#if defined( __clang__ )
+#define MR_SUPPRESS_WARNING_PUSH( x, y ) \
```

Comment:
I would add a comment that x is GCC/Clang warning id and y is MSVC warning number (or rename x and y)